src/partials/nav-tree.hbs

```diff
@@ -5,11 +5,11 @@
     {{#if ./content}}
     <div class="item {{#if (eq ./url @root.page.url)}}is-current-page{{/if}} {{#if ./items.length}}dropdown{{/if}}">
     {{#if ./url}}
-    <a class="nav-link" href="
+    <a class="nav-link {{~#if (is-beta ./url)}} beta{{~/if}}" href="
```

Comment:
maybe check for `page-cloud` and adjust the class to suit so we can make use of the `Beta` image for non-cloud docs.

Reply:
done